apps/OboeTester/app/src/main/res/layout/activity_data_paths.xml

```diff
@@ -23,18 +23,18 @@
             android:text="InPre" />
 
         <CheckBox
-            android:id="@+id/checkbox_paths_input_devices"
+            android:id="@+id/checkbox_paths_all_channels"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:checked="true"
-            android:text="InDev" />
```

Comment:
Note that we are changing parameters for automatic testing. We should update apps/OboeTester/docs/AutomatedTesting.md accordingly

Reply:
Yes. I should map the old options to the new options.
Thanks for the reminder.

Reply:
Updated docs in #1969